README.md

```diff
@@ -266,6 +267,7 @@ For those cases, additional properties (apart from `message`) are included:
     * `withCredentials` [`<Boolean>`][] - Passed to the request library to make CORS requests. **Default:** `false`
     * `payloadStructure` [`<String>`][] - (*LogDNA usage only*) Ability to specify a different payload structure for ingestion. **Default:** `default`
     * `compress` [`<Boolean>`][] - (*LogDNA usage only*) Compression support for the agent. **Default:** `false`
+    * `httpProxy` [`<String>`][] - The full URL of an http or https proxy to pass through
```

Comment:
I'm little bit concerned about naming the variable as `httpProxy` since it can be `https` proxy as well, right? Can this somehow confuse the users? Why not using just `proxy`, @darinspivey?

Reply:
Because there's no need to separate them.  A proxy is a proxy, and you can pass it http or https and it figures it out.  Less variables mean less usage confusion, IMO. The docs state that you can pass either http or https.

I guess `httpProxy` is common for other packages (env names can be HTTP_PROXY or HTTPS_PROXY), but I'm not opposed to changing it to `proxy`.  I ended up calling it that in the agent too.

Reply:
+1 for calling it `proxy`, makes sense